test/data/blurb9/_api_config.yaml

```diff
@@ -2,11 +2,9 @@ resources:
   discovery:
     path: /
     description: Provides a directory of available resources
-    actions:
-      get:
+    methods:
+      GET:
         method: GET
```

Comment:
Any reason to leave the method here? It seems from all other examples that the top key is sufficient.